django/thunderstore/api/cyberstorm/views/community_filters.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import (
+    CyberstormPackageCategorySerializer,
+    CyberstormPackageListingSectionSerializer,
+)
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.community.models import Community
+
+
+class CommunityFiltersAPIView(CyberstormAutoSchemaMixin, APIView):
+    """
+    Return info about PackageCategories and PackageListingSections so
+    they can be used as filters.
+    """
+
+    def get(self, request, community_id: str, format=None):
+        queryset = Community.objects.prefetch_related("package_categories")
+        community = get_object_or_404(queryset, identifier=community_id)
+
+        return Response(
+            {
+                "package_categories": CyberstormPackageCategorySerializer(
+                    community.package_categories.all(),
+                    many=True,
+                ).data,
+                "sections": CyberstormPackageListingSectionSerializer(
+                    community.package_listing_sections.listed().order_by("priority"),
+                    many=True,
+                ).data,
+            },
+        )
```

Comment:
If you were to create an actual response serializer class for this view specifically, this would be a bit cleaner and it could be shown in the API docs appropriately. You would probably have to ditch the use of the `CyberstormAutoSchemaMixin` in favor of the decorator + setting the serializer on it though I think.

Reply:
I guess I read your comment in an earlier PR that you're _"not against having view-specific serializers"_ in a way that you don't much prefer them either.

Also it seems `CyberstormAutoSchemaMixin` doesn't work with `APIView` classes anyway, the view didn't get classified under the cyberstorm tag.